SU2_CFD/include/solvers/CFEASolver.hpp

```diff
@@ -28,6 +28,7 @@
 #pragma once
 
 #include "CFEASolverBase.hpp"
+#include "CHeatSolver.hpp"
```

Comment:
You don't need to include CHeatSolver anymore because you are only using the virtual interface (CSolver).
```suggestion
```